src/table/__tests__/sortable-head-cell.test.js

```diff
@@ -20,6 +21,31 @@ describe('sortable-head-cell', () => {
     expect(down).toHaveLength(1);
   });
 
+  it('fillClickTarget prop enable sort on cell click', () => {
+    const spy = jest.fn();
+    const wrapper = mount(
+      <SortableHeadCell
+        onSort={spy}
+        direction="ASC"
+        title="test"
+        fillClickTarget
+      />,
+    );
+    expect(wrapper.find(StyledHeadCell).prop('$cursor')).toBe('pointer');
+    wrapper.simulate('click');
+    expect(spy).toHaveBeenCalledTimes(1);
+  });
+
+  it('without fillClickTarget prop, no sort on cell click', () => {
+    const spy = jest.fn();
+    const wrapper = mount(
+      <SortableHeadCell onSort={spy} direction="ASC" title="test" />,
+    );
+    expect(wrapper.find(StyledHeadCell).prop('$cursor')).toBeUndefined();
+    wrapper.simulate('click');
+    expect(spy).not.toHaveBeenCalled();
+  });
```

Comment:
what happens if the `disabled` prop on the component is true? does it allow clicks?

Reply:
Yes, it allow clicks. `<SortableLabel /> ` handles `disabled` prop really well but not the `<HeadCell />`. Tried to fix it with a check. Let me know what you think in latest commit.